### ISSUE TITLE
Fix for S3 breaking change from AWS

### ISSUE THREAD

**Issue by jacobredesign** [datagrok-ai/tf-module-datagrok-core#49]
Since the change of the default s3 bucket behavior (https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html) to ACLs disabled by default for all new s3 buckets, deploying datagrok core fails with "The bucket does not allow ACLs" on the logging bucket.

I believe adding a configuration of this resource https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_ownership_controls
looking something like this:
```
resource "aws_s3_bucket_ownership_controls" "bucket_acl_ownership" {
  bucket = aws_s3_bucket.chemaxon_license_bucket.bucket
  rule {
    object_ownership = "ObjectWriter"
  }
}
```

should fix the issue.

**Comment by jacobredesign** [comment]
Any update on this? This is causing our production terraform pipeline to break and it's a relatively small fix.

**Comment by dnillovna** [comment]
Hi @jacobredesign, the issue should be fixed in the latest releases. The first one to contain the fix is `2.2.1`.
